Sources/Libraries/ZincFramework/Sync/SyncSubcommand.swift

```diff
@@ -14,12 +14,12 @@ struct SyncSubcommand: ParsableCommand {
     // MARK: Options
     
     /// The Zincfile to parse.
-    @Option(name: .shortAndLong, default: "Zincfile", help: "The Zincfile to parse and use for syncing.")
-    private var file: String
+    @Option(name: .shortAndLong, help: "The Zincfile to parse and use for syncing.")
+    private var file: String = "Zincfile"
```

Comment:
Q: By removing the `default` argument, does the help page for this command now lose that information? If so, I feel like the preferred route would be to keep it as an argument in the property wrapper.

Reply:
Help page should be identical, these changes are the result of deprecations. The Property Wrapper can have access to the underlying default value, so I assume that's why they removed it and required this.